### DISCUSSION TITLE
Adding Consolas as one of the default font families

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1319] [Ideas] [answered]
It would be nice if you could use the Consolas font on Monkeytype without needing to use a custom font.

**Comment by Miodec** [comment, marked as answer]
You can just use the custom button since as far as i know consolas is available in every operating system

**Reply by unrealapex** [reply]
I'm pretty sure you need to download the font for any other operating system other than Windows.
